xml/System.Windows/ThemeMode.xml

```diff
@@ -121,9 +129,9 @@
         <Parameter Name="other" Type="System.Windows.ThemeMode" />
       </Parameters>
       <Docs>
-        <param name="other">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="other">ThemeMode object to compare with</param>
+        <summary>Checks whether the ThemeMode object is equal to another ThemeMode object.</summary>
+        <returns>Returns <see langword="true" /> when the ThemeMode objects are equal, and <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" /> if the ThemeMode objects are equal; <see langword="false" /> otherwise.</returns>
```